### PR TITLE
Fix the build errors encountered in #67

### PR DESCRIPTION
**Pull request by TobTobXX**
- [x] Add wayland socket
- [x] Share IPC
- [x] Disable automerge PRs (see [comment below](https://github.com/flathub/com.github.Anuken.Mindustry/pull/68#issuecomment-2426748847))
- [x] Fix `<ul>` tags
- [x] Add `<launchable />` tag
- [x] Fix all other appstream warnings
- [ ] Get Wayland to work.
- [ ] Test

### CODE REVIEW

**Comment by TobTobXX** [comment]
wtf is this error???
```
File "_none_", line 1:
Error: Files "debugger/ocamldebug.cmo" and "otherlibs/unix/unix.cma(Unix)"
       make inconsistent assumptions over interface "Unix"
make[2]: *** [Makefile:558: debugger/ocamldebug] Error 2
```

**Comment by TobTobXX** [comment]
bot, build. (or how was the command again?)

**Comment by TobTobXX** [comment]
Note: Automerge (in flathub.json) was disabled. Reason:

> ### flathub-json-automerge-enabled
> Exceptions allowed: Only for verified applications and extra-data apps if upstream rotates/deletes sources
>
> The flathub.json file has automerge-flathubbot-prs property enabled. This is no longer allowed by default.
-- [Flatpak Builder Lint Docs](https://docs.flathub.org/docs/for-app-authors/linter/#flathub-json-automerge-enabled)

Verification is likely out, but I don't understand the extra-data apps part.

**Comment by TobTobXX** [comment]
bot, build.

**Comment by TobTobXX** [comment]
bot, build?

**Comment by TobTobXX** [comment]
bot, abort? heeelp

File: com.github.Anuken.Mindustry.yml [hunk 1]
```diff
@@ -7,7 +7,9 @@ sdk-extensions:
 
 command: mindustry.sh
 finish-args:
+  - --socket=wayland
```
Comment (TheEvilSkeleton) [inline com.github.Anuken.Mindustry.yml:10]: Have you checked if this works?

Comment (TobTobXX) [reply com.github.Anuken.Mindustry.yml:10]: Hmm... you're right, xeyses still works for Mindustry.

Im gonna have to debug this further...

**Comment by TobTobXX** [comment]
While I'm getting wayland to work: Should I split this PR into one getting wayland to work and the other one making the flatpak buildable again? Bc right now there are many appstream issues that block flathub updates. And then I can also rebase #67 on top of master.

**Comment by TheEvilSkeleton** [comment]
Yea, that'd be for the best

**Pull request closed without merging**